L1Trigger/L1THGCal/interface/HGCalTriggerFECodecBase.h

```diff
@@ -99,7 +101,7 @@ namespace HGCalTriggerFE {
           << "Data payload was already set for HGCTriggerFECodec: "
           << this->name() << " overwriting current data!";
       }
-      static_cast<Impl&>(*this).setDataPayloadImpl(mod,ee,fh,bh);
+      static_cast<Impl&>(*this).setDataPayloadImpl(mod,geom,ee,fh,bh);
```

Comment:
If the module is properly defined we shouldn't need the geometry to do anything, no? All relevant DetIds all the way down to HGC cell level are available.